scripts/lib/libvirt/admin-config.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import libvirt
+import argparse
+import libvirt_helpers
+from string import Template
+
+conn = libvirt.open("qemu:///system")
```

Comment:
In Python 'usually' single quoted are used (except for `__doc__` ) and when the string have a single quote inside.

Feel free to ignore this, because is only a trending on top of PEP8.

https://www.python.org/dev/peps/pep-0008/#id18